.github/workflows/cygwin.yml

```diff
@@ -13,7 +13,12 @@ permissions:
   contents: read # to fetch code (actions/checkout)
 
 jobs:
+  get_commit_message:
+    name: Get commit message
+    uses: ./.github/workflows/commit_message.yml
```

Comment:
This already skips the run if not docs-only, right?  Can we encode that into the name?

In general, I think having `[docs only]` is awesome (a github label might also be, easier to add by maintainer, but a different thing).

Reply:
Agreed, the naming in SciPy is historical as an existing job was modified.

Reply:
Thought I would ping since I just saw a suggestion to add the three skips we currently need.
Not sure if there is anything else holding it up.  But, assuming it works, from my side it seems ready with that rename to clarify what the `needs:` does below.